### ISSUE TITLE
Not working vertical range annotation

### ISSUE THREAD

**Issue by xxx44yyy** [apexcharts/apexcharts.js#606]
## Codepen

[https://codepen.io/xxx44yyy/pen/PvjLvL?editors=0010](https://codepen.io/xxx44yyy/pen/PvjLvL?editors=0010)

1. vertical annotation:
![image](https://user-images.githubusercontent.com/40538167/57972282-ed088e80-79a0-11e9-80d3-8983b10ea69b.png)

2. click to child:
![image](https://user-images.githubusercontent.com/40538167/57972294-fb56aa80-79a0-11e9-986c-cff4ac8f1f7d.png)

3. see non vertical annotation:
![image](https://user-images.githubusercontent.com/40538167/57972301-0b6e8a00-79a1-11e9-8dcb-2c42f37d9ee4.png)

why green label block is non vertical? 
![image](https://user-images.githubusercontent.com/40538167/57972315-4244a000-79a1-11e9-9fc5-d722509caae1.png)

**Comment by xxx44yyy** [comment]
code:

```javascript
var options = {
  annotations: {
    xaxis: [
      {
        x: new Date("11/17/2017").getTime(),
        strokeDashArray: 0,
        borderColor: "#775DD0",
        label: {
          borderColor: "#775DD0",
          style: {
            color: "#fff",
            background: "#775DD0"
          },
          text: "X Axis Anno Vertical"
        }
      }
    ],
  },
  chart: {
    height: 380,
    type: "line",
    id: "areachart-2",
        toolbar: {
    show: true,
    tools: {
      download: false,
      selection: false,
      zoom: false,
      zoomin: false,
      zoomout: false,
      pan: false,
      reset: false,
      customIcons: [{
 icon: '<img src="http://adecinc.com/wp-content/uploads/2018/07/baby-baby-girl-newborn-raw-simple-icon-png-18.png" width="16">',
 title: 'xxxx',
 click: function() {
 chart.addXaxisAnnotation({
   x: new Date('26 Nov 2017').getTime(),
   x2: new Date('28 Nov 2017').getTime(),
   fillColor: '#B3F7CA',
   opacity: 0.4,
   label: {
       borderColor: '#B3F7CA',
       style: {
           fontSize: '10px',
           color: '#fff',
           background: '#00E396',
       },
       offsetY: -10,
     text: 'X-axis range'
   }
 });
 }
                              }]
                          }
                      }
  },
  series: [
    {
      data: series.monthDataSeries1.prices
    }
  ],
  labels: series.monthDataSeries1.dates,
  xaxis: {
    type: "datetime"
  },
};

var chart = new ApexCharts(document.querySelector("#chart"), options);

chart.render();


```

**Comment by junedchhipa** [comment]
Fixed the issue.
Will be released in 3.6.12

**Comment by xxx44yyy** [comment]
@junedchhipa Thanks!! How can I use your changes right now? I did not work with webpack, react, vue. I can't find min.js

**Comment by xxx44yyy** [comment]
I'm use only this file: https://cdn.jsdelivr.net/npm/apexcharts@latest

**Comment by junedchhipa** [comment]
The new patch version will be released tomorrow. Please use the same cdn link when it is released as it will be automatically updated too.

**Comment by xxx44yyy** [comment]
ok :(

**Comment by dekanaxo** [comment]
The new patch version will be released tomorrow. Please use the same cdn link when it is released as it will be automatically updated too.


How soon?

**Comment by xxx44yyy** [comment]
> How soon?

I don't know. :( Now 3.6.10

@junedchhipa

**Comment by junedchhipa** [comment]
Released 3.6.12
https://cdn.jsdelivr.net/npm/apexcharts@latest
https://unpkg.com/apexcharts@3.6.12/dist/apexcharts.min.js

The CDN might take some hours to propagate the changes.

**Comment by xxx44yyy** [comment]
> Released 3.6.12
> https://cdn.jsdelivr.net/npm/apexcharts@latest
> https://unpkg.com/apexcharts@3.6.12/dist/apexcharts.min.js
> 
> The CDN might take some hours to propagate the changes.

Thanks!!!!!
